index.js

```diff
@@ -145,8 +143,70 @@ const setSuggestion = (result) => {
   document.getElementById("suggestion").innerText = result;
 };
 
-const filterBtn = document.getElementById("filter-btn");
-filterBtn.onclick = () => {
+const onClick = () => {
   clearAll("suggestion");
   generateWord();
 };
+
+const filterBtn = document.getElementById("filter-btn");
+filterBtn.onclick = onClick;
+
+const ENTER_CODE = "Enter";
+
+document.addEventListener("keydown", (e) => {
+  if (e.key == ENTER_CODE) {
+    onClick();
+  }
+});
+
+const COUNT_WEIGHT = 0.5;
+const POS_WEIGHT = 1;
+const NO_DUPLICATE_CHAR = true;
```

Comment:
How it's different from the checkbox value?

Reply:
does not add weight for same char in words. Like "saab" => 5 + 10 +0 +15 (ignores 2nd "a" weight). Renamed to "SKIP_DUPLICATE_CHAR_WEIGHT".